packages/gatsby-plugin-netlify-cms/src/cms.js

```diff
@@ -1,8 +1,6 @@
-/* eslint-disable no-unused-vars */
-/* eslint-env browser */
 import CMS from "netlify-cms"
 import "netlify-cms/dist/cms.css"
-import netlifyIdentityWidget from "netlify-identity-widget"
 
-window.netlifyIdentity = netlifyIdentityWidget
-netlifyIdentityWidget.init()
+if (NETLIFY_CMS_PREVIEW_STYLES_SET) {
```

Comment:
The CI builds are failing because this is an undefined variable.

Reply:
Ah, I see that now. Weird that it passes linting locally.